plugins/kubernetes-backend/src/service/KubernetesFanOutHandler.ts

```diff
@@ -211,19 +211,14 @@ export class KubernetesFanOutHandler {
     entity,
     auth,
   }: KubernetesObjectsByEntity): Promise<ObjectsByEntityResponse> {
-    return this.fanOutRequests(
-      entity,
-      auth,
-      this.objectTypesToFetch,
-      this.customResources,
-    );
+    return this.fanOutRequests(entity, auth, this.objectTypesToFetch, null);
   }
 
   private async fanOutRequests(
     entity: Entity,
     auth: KubernetesRequestAuth,
     objectTypesToFetch: Set<ObjectToFetch>,
-    customResources: CustomResourceMatcher[],
+    customResources: CustomResourceMatcher[] | null,
```

Comment:
Parameter can be made optional and omitted on line 214

Reply:
`fanOutRequests` function is invoked by `getCustomResourcesByEntity` and  `getKubernetesObjectsByEntity`. 

When `getCustomResourcesByEntity` invokes `fanOutRequests`, we need to make sure that we use the customResources parameter because it was provided in the [HTTP request](https://github.com/andmagom/backstage/blob/f24ce343eba13b7250fbf471aa52762646f3dcdc/plugins/kubernetes-backend/src/routes/resourcesRoutes.ts#L76).

When `getKubernetesObjectsByEntity` invokes `fanOutRequests`, we can use the customResources defined at cluster level or the customResources defined globally, taking precedence the cluster level.

Because we can not know who is calling the fanOutRequests function, the null parameter let's take decisions to use the customResources datas in a [specific order](https://github.com/andmagom/backstage/blob/f24ce343eba13b7250fbf471aa52762646f3dcdc/plugins/kubernetes-backend/src/service/KubernetesFanOutHandler.ts#L253).

Reply:
Hello @mclarke47, Talking with @jamieklassen, I have changed the code to use optional parameter on line 221. Let me know if it is the change that you are looking for, then I can start to modify the docs !